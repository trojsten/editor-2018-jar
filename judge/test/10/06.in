INT:
velkostmapy
7
INT:
dlzka
0
INT:
vyska
0
INT:
okuliare
0
VECTOR_STR:
mapa
7
#######
#O$YQ[#
#=V7)L#
#$5/(+#
#0R2[.#
#######
E6XNP>8
