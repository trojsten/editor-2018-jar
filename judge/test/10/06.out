{"dlzka": 7, "vyska": 6}